.github/workflows/security.yml

```diff
@@ -3,8 +3,20 @@ name: Security Checks
 on:
   push:
     branches: [ main ]
+    paths-ignore:
+      - '.github/**'
```

Comment:
Let's keep github workflow related files as part of the security checks.

Also not ignore .slsa-goreleaser maybe (thoughts @anoop2811 )?